src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -156,6 +156,13 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit, AfterViewIni
 
   public tabIndex: number = -1;
 
+  /**
+   * Specifies a familiar object for this repeater item to be returned when a `orderChange` event fires.
```

Comment:
I'd delete "familiar." And I'd tweak the wording to use active voice: "Specifies an object that the repeater component returns for this repeater item when the `orderChange` event fires."

Reply:
Done.